### DISCUSSION TITLE
Installing scoop again after HDD died

### DISCUSSION THREAD

**Discussion by airworksman** [ScoopInstaller/Scoop#5443] [Help and Questions] [answered]
Hi, I recently had my old secondary HDD that had scoop on it die. I didn't have much on it so I'm planning on just installing from scratch to the same location (D:\scoop). I'm just wondering if there is anything I need to do beforehand or anything I should be aware of. The most obvious is the dead shortcuts left behind, do I need to remove these or I'm assuming when I reinstall all the same apps they will overwrite these so it's not necessary? Similarly if any apps made registry edits, they will just overwrite the old ones so nothing needs to be done here either?

Anything else I may be missing? Thanks.

**Comment by rashil2000** [comment, marked as answer]
>  I'm assuming when I reinstall all the same apps they will overwrite these so it's not necessary? Similarly if any apps made registry edits, they will just overwrite the old ones so nothing needs to be done here either?

Yes, they will be overwritten, so it should be fine.
